client/src/containers/SampleContainer/SampleContainer.js

```diff
@@ -16,6 +17,7 @@ class SampleContainer extends PureComponent {
     return (
       <div className="SampleContainer">
         <SampleComponent message={this.props.sample.message} />
+        <HowToPlay />
```

Comment:
Let's not use SampleContainer to display HowToPlay Component. This belongs in a separate page and a url.

Reply:
I will change it and make pr by tonight